sdk/src/feature_set.rs

```diff
@@ -49,6 +49,15 @@ pub mod full_inflation {
         }
     }
 
+    pub mod stakeconomy {
+        pub mod vote {
+             solana_sdk::declare_id!("JCergKv4GcywaBzn4JHi3sYJfG7mWenTG3QQDNUJiGS");
```

Comment:
Hey @stakeconomy, the code beautifier is complaining here.  Can you please make this edit:
```suggestion
            solana_sdk::declare_id!("JCergKv4GcywaBzn4JHi3sYJfG7mWenTG3QQDNUJiGS");
```

Reply:
Oh, looks like github let me do it.  So fixed :)